scss/styles.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
```

Comment:
I think you have more than 2 spaces here!

Reply:
Yes. I think same!